airlock/templates/file_browser/file.html

```diff
@@ -43,45 +43,91 @@
+            <form action="{{ file_reset_review_url }}" method="POST">
+              {% csrf_token %}
+              <button aria-pressed="true" class="btn-group__btn btn-group__btn--left btn-group__btn--active" id="file-reset-button" type="submit">
+                Approve file
+              </button>
+            </form>
+          {% endif %}
 
-        {% if file_reject_url %}
-          <form action="{{ file_reject_url }}" method="POST">
-            {% csrf_token %}
-            {% #button type="submit" variant="danger" href=file_reject_url id="file-reject-button" %}Request Changes{% /button %}
-          </form>
-        {% else %}
-          {% #button type="submit" variant="danger" disabled=True href=file_reject_url id="file-reject-button" tooltip="File changes already requested." %}Request Changes{% /button %}
-        {% endif %}
-        {% #button variant="primary" type="link" href=path_item.download_url id="download-button" %}Download file{% /button %}
+          {% if file_reject_url %}
+            <form action="{{ file_reject_url }}" method="POST">
+              {% csrf_token %}
+              <button aria-pressed="false" class="btn-group__btn btn-group__btn--right" id="file-reject-button" type="submit">
+                Request changes
+              </button>
+            </form>
+          {% elif file_reset_review_url and file_approve_url %}
+            <form action="{{ file_reset_review_url }}" method="POST">
+              {% csrf_token %}
+              <button aria-pressed="true" class="btn-group__btn btn-group__btn--right btn-group__btn--active" id="file-reset-button" type="submit">
+                Request changes
+              </button>
+            </form>
+          {% endif %}
+        </div>
       {% endif %}
     {% endif %}
-    {% #button variant="primary" type="link" href=path_item.contents_url external=True id="view-button" %}View ↗{% /button %}
-    {% if code_url %}
-      {% #button type="link" variant="primary" id="file-code-button" href=code_url external=True %}View code{% /button %}
-    {% endif %}
-  </div>
 
+    <details-utils close-click-outside close-esc>
+      <details class="more-dropdown">
+        <summary class="more-dropdown__btn" id="file-button-more">
+          More…
+          <span class="opacity-75 ml-1">
+            <img height="16" width="16" class="icon inline-block group-open:hidden" src="/static/icons/keyboard_arrow_down.svg" alt="">
+            <img height="16" width="16" class="icon hidden group-open:inline-block" src="/static/icons/keyboard_arrow_up.svg" alt="">
+          </span>
+        </summary>
+        <ul class="more-dropdown__nav">
+          {% if is_output_checker %}
+            <li class="more-dropdown__item">
+              <a href="{{ path_item.download_url }}" class="more-dropdown__link" id="download-button">
+                <img class="more-dropdown__icon" src="/static/icons/cloud_download.svg" alt="">
+                Download file
```

Comment:
This makes the download option visible from a workspace view for all output-checkers. It should (currently at least) only be allowed for a file in a release request, and only for output checkers if they are not the author of the request.
(For release requests, the download url 403s for an author who's also output checker; for workspace files the method that generates the `path_item.download_url` just ignores the download bit and returns the content url, so it doe sthe same as the "View file" button)

Reply:
I think I've covered that off now with:

```py
{% if is_output_checker and context == "request" and not is_author %}
```

Is that all that is required?

Reply:
I think `{% if context == "request" and not is_author %}` is enough - a user shouldn't be able to see the request at all unless  they're an output checker or the author

Reply:
Yes, `get_release_request_or_raise` is called for every request-related view, and it'll raise a 403 if the user isn't either an output-checker or the author

Reply:
👌 perfect, I'll change that